app/utils/httpRequests.ts

```diff
@@ -19,19 +24,47 @@ export async function httpGet<T>(
     return window.http.get<T>(urlString, params);
 }
 
-export async function getTransactions(address: string, id = '0') {
-    return window.http.getTransactions(address, id);
-}
-
 export async function gtuDrop(address: string) {
     return window.http.gtuDrop(address);
 }
 
-export async function getNewestTransactions(
+export async function getTransactionsAscending(
+    address: string,
+    transactionFilter: TransactionFilter,
+    limit: number,
+    id?: string
+) {
+    const response = await window.http.getTransactions(
+        address,
+        transactionFilter,
+        limit,
+        TransactionOrder.Ascending,
+        id
+    );
+
+    return {
+        transactions: response.transactions,
+        full: response.full,
+    };
```

Comment:
Is there any reason we are de/reconstructing the output in this and getTransactionsDescending?
```suggestion
    return window.http.getTransactions(
        address,
        transactionFilter,
        limit,
        TransactionOrder.Ascending,
        id
    );
```